libr/reg/double.c

```diff
@@ -94,7 +97,10 @@ R_API bool r_reg_set_longdouble(RReg *reg, RRegItem *item, long double value) {
 	case 80:
 	case 96:
 	case 128:
-		r_mem_copyendian ((ut8 *)&vld, (ut8 *)&value, 10, !reg->big_endian);
+		if (0 /* XXX which endian? */)
```

Comment:
i think >64 regs are only handled as packed values so they are in fact 4 LE words